icdiff

```diff
@@ -740,7 +741,7 @@ def diff_files(options, a, b):
             else:
                 codec_print("error: to use arbitrary file labels, "
                             "specify -L twice.", options)
-                return diff_found
+                sys.exit(EXIT_CODE_ERROR)
```

Comment:
Same here.  That looked more like an error to me than something that is only printing a message, but then does not let icdiff stop.